deployments/kubernetes/chart/reloader/templates/deployment.yaml

```diff
@@ -76,8 +76,15 @@ spec:
       - image: "{{ .Values.reloader.deployment.image.name }}:{{ .Values.reloader.deployment.image.tag }}"
         imagePullPolicy: {{ .Values.reloader.deployment.image.pullPolicy }}
         name: {{ template "reloader-fullname" . }}
-      {{- if or (.Values.reloader.deployment.env.open) (.Values.reloader.deployment.env.secret) (.Values.reloader.deployment.env.field) (.Values.reloader.deployment.env.existing) (eq .Values.reloader.watchGlobally false) (.Values.reloader.enableHA) (.Values.reloader.enableMetricsByNamespace)}}
```

Comment:
If the code is removed and none of these values is defined you'll get and empty env:
```
env:
```
Which is legal yaml and k8s will handle it as expected.

Reply:
Empty vars is fine and will be handled. But since we have these defined in values.yaml, its better to have conditional checks instead and not have the empty var at all :)